src/spootnik/reporter/sentry.clj

```diff
@@ -1,10 +1,11 @@
 (ns spootnik.reporter.sentry
   (:require
-   [sentry-clj.core :as sentry-io]
-   [manifold.deferred  :as d]
-   [clojure.string :as str]
-   [clojure.tools.logging :refer [error]]
-   [clojure.java.shell :as sh]))
+    [sentry-clj.core :as sentry]
+    [sentry-clj.core :as sentry-io]
+    [manifold.deferred :as d]
+    [clojure.string :as str]
+    [clojure.tools.logging :refer [error]]
+    [clojure.java.shell :as sh]))
```

Comment:
Using one name for the same import in order to prevent confusion:
```suggestion
    [sentry-clj.core :as sentry]
    [manifold.deferred :as d]
    [clojure.string :as str]
    [clojure.tools.logging :refer [error]]
    [clojure.java.shell :as sh]))
```

Reference have to be updated accordingly in the rest of the package.